app/styles/app.css

```diff
@@ -1,4 +1,5 @@
 /* Ember supports plain CSS out of the box. More info: https://cli.emberjs.com/release/advanced-use/stylesheets/ */
+@layer utilities, scoped-components;
```

Comment:
The layers order to be defined directly in `app/index.html`, which was an oddity of ember-scoped-css in classic. When using Vite, we can define this more intuitively in `app.css`.